### DISCUSSION TITLE
Backup Config is exporting channels and alerts, not only config

### DISCUSSION THREAD

**Discussion by ppazos** [nextgenhealthcare/connect#6126] [Q&A]
I did a backup and a restore of the settings, then when I was about to import my alerts, the alerts where already there!

Checking my steps, detected the file generated for the Backup Config actually had everything: server config, channels, alerts, etc.

While having one file with everything is useful, I'm wondering if Mirth should be more clear about that backing up the config will actually backup everything.

On the side, it would be desirable to have one functionality that just exports the config without everything else, because then restoring the config seems to overwrite everything on the server without letting the admin know (for instance the alert was created without noticing me). So if I want to import just certain channels and then the config, it will overwrite those imports and import channels I don't want.

**Comment by pacmano1** [comment]
See related feature already requested https://github.com/nextgenhealthcare/connect/issues/5654
